packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -98,6 +106,7 @@ function connect() {
       // Device was already connected
       resolve();
     }
+    resolve();
```

Comment:
Remove this line. This `resolve` will get called before the `resolve` on line 94.